src/api/app/models/event/added_user_to_group.rb

```diff
@@ -0,0 +1,11 @@
+module Event
+  class AddedUserToGroup < Base
+    self.message_bus_routing_key = 'groups_user.create'
```

Comment:
I'm don't think this should go over the message bus. It's a personal notification, not a public one.

Reply:
Aha!  I'm slightly unaware of this; I copied it from an example I was looking at.  Is there documentation on the difference that I missed?

Reply:
https://github.com/openSUSE/open-build-service/wiki/Events#publishing-events-to-the-rabbitmq-message-bus